### ISSUE TITLE
Custom crafting recipe disables crafting of other items

### ISSUE THREAD

**Issue by airfighter001** [AppliedEnergistics/Applied-Energistics-2#2314]
Changed the crafting recipe of certus quartz, nether quartz and fluix seeds to give the final pure crystals as the energy acceptor wouldn't accept the energy I could provide and in my case waiting for the seeds seemed lika a waste of time anyways.
After this I wasn't able to craft the ME controller or the Inscriber. The recipe showed up in NEi fine (after I figured that recipe mode for AE2-items only works with the changed recipe in the server config but not on the client) and the item showed as result in the crafting table. However, it isn't possible to take the item out of the table, it just jumps right back to the crafting result slot. Neither me nor another person on my server was able to take it out, normal click, shift click, auto workbench, nothing worked.

Since the last change I made to the server before this happened was the custom recipe I removed this with the result me being able to craft again.
The custom recipe itself worked well btw after we figured how to do it corretly (not very well documented imho)

Tl;dr:
Adding custom crafting recipe -> crafting of the tested AE2 items isn't possible anymore but the custom recipe works

/edit: Forgot to mention: I use appliedenergistics2-rv3-beta-6

**Comment by yueh** [comment]
That is even described it the bundled readme in the recipes folder.
